instagram/src/components/Authentication/Authenticate.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Login from '../Login/Login';
+import './Authenticate.css';
+
+
+const Authenticate = PassedComponent =>
+  class extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = {loggedIN: false}
+    }
+
+    componentDidMount() {
+      if (localStorage.getItem('user' === 'ImaHacker') &&
```

Comment:
This is the part of your Authentication that is breaking. If you only if a user key exists in the localStorage instead of checking the value, it will let you login. Also on line 15 you have a misplaced parenthesis right after password that should wrap the whole statement